ci/build-in-docker.sh

```diff
@@ -74,7 +75,9 @@ echo "Requires.private: libcares libnghttp2 libidn2 libssl openssl libcrypto lib
 # in a Docker container, we can safely disable this check
 git config --global --add safe.directory '*'
 
+bash -euxo pipefail /source/ci/install-static-desktop-file-validate.sh 56d220dd679c7c3a8f995a41a27a7d6f3df49dea
```

Comment:
This would have to be updated constantly. We need to set up Dependabot or so to keep it up to date. There is no regime that hardcode-pins a specific release of the XDG desktop entry spec (that would need to be implied by the spec), nor is there a good reason to reject AppImages that use a newer spec (they might run on newer distros just fine or the new keys might fix problems, why reject that).

Reply:
What's the reason for pinning a specific commit rather than a tag?

Reply:
Please change to [0.28](https://gitlab.freedesktop.org/xdg/desktop-file-utils/-/tags/0.28) (latest stable release) and try again.

Reply:
Seems like 56d220dd679c7c3a8f995a41a27a7d6f3df49dea added  support for spec 1.5. We should pin a release, though.